content/blog/2024-11-12-ecl80/index.md

```diff
@@ -0,0 +1,172 @@
+
+“As more organizations and governments discover the potential of the CHERI technology to protect us, we need to speed up the pace of making the technology available in real systems," says Codasip’s chief executive officer Ron Black of his company’s largesse. “We have made a massive effort to implement a full Linux-capable SDK that we are now opening for everyone to use. I am confident this will be a great asset for the CHERI and RISC-V communities.”
+
+“The CHERI Alliance is strongly focused on collaboration and openness to make sure that CHERI security gets integrated into all high-tech products,” adds CHERI Alliance director Michael Halsall. “The fact that Codasip makes their SDK openly available through the Alliance supports the standardisation effort of CHERI for RISC-V. CHERI can deliver a more secure future for electronics, and we must come together to make that happen, between academia, industry and government.”
+
+The SDK is available [from the CHERI Alliance GitHub organisation](https://github.com/CHERI-Alliance), under a range of free and open-source licences.
+
+<img src="calistaredmond.jpg" style="max-width:100%" />
+
+## RISC-V’s Calista Redmond Highlights the “Stronger Path from Research to Commercialisation”
+
+  
+RISC-V International chief executive officer Calista Redmond espoused the benefits of the recently-passed European Chips Act in providing “a much stronger path from research to commercialisation” of RISC-V designs, in an interview with eeNews Europe’s Nick Flaherty.
+
+“When the community comes together you get a boost, you get the proof points,” Calista tells Nick in the interview. "In the past there was stronger reliance on collaborative research in different labs and these now have acceleration to put more confidence into those projects and outcomes. We have organisations coming together for commercialisation such as Quintauris, you have organisations like OpenCHIP, the more stakeholders you bring into the tent the more you have to satisfy and align their requirements.
+
+“But we get that this means RISC-V is the broader global incubator that demonstrates the wider approach to investors and governments," Calista notes. “We have 80 workgroups and committees coming together and we have started a certification committee that is coming together, not just for the base ISA but for platforms.”
+
+Other topics touched on in the interview include RISC-V International’s relationship with those deploying the ISA in China - “we don’t engage in commercial developments,” she says, “we provide the composable building blocks” - the status of RISC-V as a global standard, the organisation’s approach to security, and growth in the software ecosystem.
+
+The full interview is available [on eeNews Europe now](https://www.eenewseurope.com/en/ceo-interview-chips-act-boost-for-risc-v/).
+
+<img src="kairos.jpg" style="max-width:100%" />
```

Comment:
This file is missing.

Reply:
Weird - mustn't have copied across from my temporary folder. Added!